test/waitforapp_test.js

```diff
@@ -1,10 +1,41 @@
 suite('waitforapp', function() {
   // requires
+  var launch = require('../lib/launch').launch;
+  var switchToApp = require('../lib/switchtoapp').switchToApp;
   var waitForApp = require('../lib/waitforapp').waitForApp;
+  var waitForActivity = require('../lib/waitforapp').waitForActivity;
+
+  var ACTIVITY_CALLER_DOMAIN = 'activity.caller.gaiamobile.org';
+  var ACTIVITY_CALLEE_DOMAIN = 'activity.callee.gaiamobile.org';
+
+  var apps = {};
+  apps[ACTIVITY_CALLER_DOMAIN] = __dirname + '/apps/activitycaller';
+  apps[ACTIVITY_CALLEE_DOMAIN] = __dirname + '/apps/activitycallee';
+
+  var client = createClient({
+    settings: {
+      'ftu.manifestURL': null,
+      'lockscreen.enabled': false,
+      'screen.timeout': 0
+    },
+
+    apps: apps
+  });
 
-  var client = createClient();
   marionette.plugin('mozApps', require('../lib/apps'));
 
+  function getParentClass(iframeElement) {
+    return iframeElement.scriptWith(function(el) {
```

Comment:
I changed this and several things below to ```sync``` syntax (without callbacks), but maybe I'm missing something and we should use ```callback``` syntax?